script/index.js

```diff
@@ -0,0 +1,138 @@
+
+let myLibrary = [];
+
+
+function Book(title, author, numberOfPages, read) {
+  this.title = title;
+  this.author = author;
+  this.numberOfPages = numberOfPages;
+  this.read = read;
+}
+
+function getBookValues() {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const numberOfpages = document.getElementById('numberOfpages').value;
+  const read = document.getElementById('read').value;
+  const book = new Book(title, author, numberOfpages, read);
+  return book;
+}
+
+function showBooks() {
+  const tableBody = document.querySelector('.books-list');
+  tableBody.innerHTML = '';
+  if (myLibrary.length > 0) {
+    for (let i = 0; i < myLibrary.length; i += 1) {
+      const row = `
+          <tr id="book-${i}"> 
+              <td>${myLibrary[i].title} </td> 
+              <td>${myLibrary[i].author} </td>
+              <td>${myLibrary[i].numberOfPages} </td>
+              <td><input ${myLibrary[i].read === 'true' ? 'checked' : ''} type="checkbox" id="read" onclick = "update(${i})"/></td>
```

Comment:
- [ ] For every function apart from the Book constructor function kindly make them es6 function syntax.